gcloud-java-examples/src/main/java/com/google/gcloud/examples/Stat.java

```diff
@@ -0,0 +1,104 @@
+package com.google.gcloud.examples;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.spi.FileSystemProvider;
+
+/**
+ * Stat is a super-simple program that just displays the size of the file
+ * passed as argument.
+ *
+ * <p>It's meant to be used to test GCloud's integration with Java NIO.
+ *
+ * <p>You can either use the '--check' argument to see whether GCS is enabled,
+ * or you can directly pass in a GCS file name to use. In that case you have to
+ * be logged in (using e.g. the gcloud auth command).
+ *
+ * <p>See the README for a command line to run this example.
+ */
+public class Stat {
+
+  /**
+   * See the class documentation.
+   */
+  public static void main(String[] args) throws IOException {
+    if (args.length == 0 || args[0].equals("--help")) {
+      help();
+      return;
+    }
+    if (args[0].equals("--list")) {
+      listFilesystems();
+      return;
+    }
+    if (args[0].equals("--check")) {
+      checkGcs();
+      return;
+    }
+    for (String a : args) {
+      statFile(a);
+    }
+  }
+
+  /**
+   * Print the length of the indicated file.
+   *
+   * <p>This uses the normal Java NIO Api, so it can take advantage of any installed
+   * NIO Filesystem provider without any extra effort.
+   */
+  private static void statFile(String fname) {
+    try {
+      Path path = Paths.get(new URI(fname));
+      long size = Files.size(path);
+      System.out.println(fname + ": " + size + " bytes.");
+    } catch (Exception ex) {
+      System.out.println(fname + ": " + ex.toString());
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://www.codacy.com/app/mziccard/gcloud-java/file/2307979954/issues/source?bid=3123107&fileBranchId=3162887#l59)